### ISSUE TITLE
Second run of `pnpm install --shinkwrap-only` is very slow

### ISSUE THREAD

**Issue by rarkins** [pnpm/pnpm#992]
### pnpm version:

1.29.1

### Code to reproduce the issue:

```
mkdir testpnpm
cd testpnpm
wget https://raw.githubusercontent.com/pnpm/pnpm/master/package.json
pnpm install --shrinkwrap-only
pnpm install --shrinkwrap-only
```

### Expected behavior:

The second run should be same or faster than the first.

### Actual behavior:

The first `pnpm` (i.e. generating the shrinkwrap.yaml):
```
real	0m7.778s
user	0m6.103s
sys	0m2.156s
```

The second run:
```
real	0m36.752s
user	0m23.042s
sys	0m17.472s
``` 

The above is repeatable every time I try it. All subsequent runs are just as slow too. Deleting `shrinkwrap.yaml` results in the time dropping back down again for the next install.

**Comment by zkochan** [comment]
This is happening because when shrinkwrap.yaml is available, pnpm downloads the package tarballs to the store. It is a bug

**Comment by zkochan** [comment]
fixed in `pnpm@1.32.1`
